_posts/2019-12-13-cockpit-209.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cockpit 209
+author: mvo
+date: 2019-12-13
+tags: cockpit linux
+slug: cockpit-209
+category: release
+summary: Cockpit with Overview improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 209.
+
+### A new design for the Overview page
+
+The landing page has been completely redesigned:
+
+![Overview](/images/new-overview.png)
+
+### Session timeouts
+
+Cockpit will automatically log you out of your current session after
+15 minutes of inactivity.
+
+![Automatic logout](/images/session-timeout.png)
+
+You can configure the timeout in `/etc/cockpit/cockpit.conf`.  For
+example, this will disable automatic logouts completely:
+```
+[Session]
+IdleTimeout=0
+```
+
+### Support for banners on the login page
+
+Like SSH, Cockpit can now optionally show the content of banner files
+on the login screen.
+
+![/etc/issue](/images/etc-issue.png)
+
+This need to be configured in `/etc/cockpit/cockpit.conf`.  For example:
+```
+[Session]
+Banner=/etc/issue.cockpit
+```
+
+### Support for TLS client certificate authentication
+
+You can now log into Cockpit with a TLS client certificate.
```

Comment:
, provided by the browser or e. g. a Smart Card, YubiKey, or similar device. Let's link to the documentation, too: https://cockpit-project.org/guide/latest/cert-authentication.html

Reply:
Done.